### DISCUSSION TITLE
Capture screenshot of the current page

### DISCUSSION THREAD

**Discussion by justoverclockl** [BetaHuhn/feedback-js#53] [Ideas] [answered]
Will be great if at every feedback your app can send a screenshot of the website, can be used to capture a bug for example.

**Comment by BetaHuhn** [comment, marked as answer]
Although this would be a cool feature, I don't think it's in the current scope of `feedback-js`. It would require a lot of changes and additional work which I am not willing to do/maintain right now, sorry!

I am keeping this in mind and might come back to it later though.

**Reply by justoverclockl** [reply]
not necessairly lot of work, you can use http://html2canvas.hertzen.com/

**Reply by BetaHuhn** [reply]
I already know of `html2canvas`, capturing the image itself is not the problem. 

Right now `feedback-js` is quite simple and easy to integrate with different backends/services as a user. Sending/uploading an image is always a bit more complicated and would make `feedback-js` harder to use and for me to maintain. Adding a new feature always means more work for the maintainer on the long term.

Currently I don't have enough time to implement this myself, but if you would be willing to create a good PR for this I might reconsider it.
